[2023-06-22 13:27:20,541] INFO Terminating process due to signal SIGTERM (org.apache.kafka.common.utils.LoggingSignalHandler)
[2023-06-22 13:27:20,648] INFO [KafkaServer id=0] shutting down (kafka.server.KafkaServer)
[2023-06-22 13:27:20,728] INFO [KafkaServer id=0] Starting controlled shutdown (kafka.server.KafkaServer)
[2023-06-22 13:27:20,799] WARN [KafkaServer id=0] Error connecting to node 192.168.8.103:9092 (id: 0 rack: null) (org.apache.kafka.clients.NetworkClient)
java.net.SocketException: Network is unreachable
	at java.base/sun.nio.ch.Net.connect0(Native Method)
	at java.base/sun.nio.ch.Net.connect(Net.java:483)
	at java.base/sun.nio.ch.Net.connect(Net.java:472)
	at java.base/sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:692)
	at org.apache.kafka.common.network.Selector.doConnect(Selector.java:277)
	at org.apache.kafka.common.network.Selector.connect(Selector.java:255)
	at org.apache.kafka.clients.NetworkClient.initiateConnect(NetworkClient.java:977)
	at org.apache.kafka.clients.NetworkClient.ready(NetworkClient.java:301)
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:65)
	at kafka.server.KafkaServer.doControlledShutdown$1(KafkaServer.scala:581)
	at kafka.server.KafkaServer.controlledShutdown(KafkaServer.scala:638)
	at kafka.server.KafkaServer.$anonfun$shutdown$2(KafkaServer.scala:660)
	at kafka.utils.CoreUtils$.swallow(CoreUtils.scala:68)
	at kafka.server.KafkaServer.shutdown(KafkaServer.scala:660)
	at kafka.Kafka$.$anonfun$main$3(Kafka.scala:100)
	at kafka.utils.Exit$.$anonfun$addShutdownHook$1(Exit.scala:38)
	at java.base/java.lang.Thread.run(Thread.java:829)
[2023-06-22 13:27:21,102] WARN [KafkaServer id=0] Error during controlled shutdown, possibly because leader movement took longer than the configured controller.socket.timeout.ms and/or request.timeout.ms: Connection to 192.168.8.103:9092 (id: 0 rack: null) failed. (kafka.server.KafkaServer)
[2023-06-22 13:27:21,295] WARN Session 0x10019786a6f0000 for sever localhost/127.0.0.1:2181, Closing socket connection. Attempting reconnect except it is a SessionExpiredException. (org.apache.zookeeper.ClientCnxn)
EndOfStreamException: Unable to read additional data from server sessionid 0x10019786a6f0000, likely server has closed socket
	at org.apache.zookeeper.ClientCnxnSocketNIO.doIO(ClientCnxnSocketNIO.java:77)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:350)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1290)
[2023-06-22 13:27:22,973] INFO Opening socket connection to server localhost/127.0.0.1:2181. (org.apache.zookeeper.ClientCnxn)
[2023-06-22 13:27:22,973] INFO SASL config status: Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2023-06-22 13:27:22,975] WARN Session 0x10019786a6f0000 for sever localhost/127.0.0.1:2181, Closing socket connection. Attempting reconnect except it is a SessionExpiredException. (org.apache.zookeeper.ClientCnxn)
java.net.ConnectException: Connection refused
	at java.base/sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at java.base/sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:777)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:344)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1290)
